assets/dev/js/frontend/handlers/video.js

```diff
@@ -66,10 +71,79 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 		this.getLightBox().setVideoAspectRatio( this.getElementSettings( 'aspect_ratio' ) );
 	}
 
+	prepareYTVideo( YT, onOverlayClick ) {
+		const elementSettings = this.getElementSettings(),
+			playerOptions = {
+				videoId: this.videoID,
+				events: {
+					onReady: () => {
+						if ( elementSettings.mute ) {
+							this.player.mute();
+						}
+
+						if ( elementSettings.autoplay || onOverlayClick ) {
+							this.player.playVideo();
+						}
+					},
+					onStateChange: ( event ) => {
+						if ( event.data === YT.PlayerState.ENDED && elementSettings.loop ) {
+							this.player.seekTo( elementSettings.start || 0 );
+						}
+					},
+				},
+				playerVars: {
+					controls: elementSettings.controls ? 1 : 0,
```

Comment:
Here and next vars
```suggestion
					controls: ! ! elementSettings.controls,
```

Reply:
According to [YouTube API](https://developers.google.com/youtube/player_parameters.html?playerVersion=HTML5)